tests/test_examples.py

```diff
@@ -101,8 +101,7 @@
     'sparse_gamma_def.py --num-epochs=2 --eval-particles=2 --eval-frequency=1 --guide easy',
     'svi_horovod.py --num-epochs=2 --size=400 --no-horovod',
     'toy_mixture_model_discrete_enumeration.py  --num-steps=1',
-    xfail_param('sparse_regression.py --num-steps=2 --num-data=50 --num-dimensions 20',
-                reason='https://github.com/pyro-ppl/pyro/issues/2082'),
+    'sparse_regression.py --num-steps=100 --num-data=100 --num-dimensions 11',
```

Comment:
note that this test takes ~3 seconds

Reply:
Thanks for fixing the sparse regression test!